package.json

```
@@ -21,20 +21,20 @@
     "test:unit:coverage": "DEBUG= nyc --reporter=text-summary --reporter=lcov mocha",
     "test:functional": "npm run test:functional:legacy && npm run test:functional:sdk",
     "test:functional:legacy": "npm run test:functional:legacy:http && npm run test:functional:legacy:websocket && npm run test:functional:legacy:mqtt && npm run test:functional:legacy:cluster",
-    "test:functional:legacy:http": "npx cucumber-js --format progress-bar --profile http",
-    "test:functional:legacy:websocket": "npx cucumber-js --format progress-bar --profile websocket",
-    "test:functional:legacy:mqtt": "npx cucumber-js --format progress-bar --profile mqtt",
-    "test:functional:legacy:cluster": "npx cucumber-js --format progress-bar --profile websocketCluster",
+    "test:functional:legacy:http": "npx cucumber-js --format progress-bar --profile http ./features-legacy",
+    "test:functional:legacy:websocket": "npx cucumber-js --format progress-bar --profile websocket ./features-legacy",
+    "test:functional:legacy:mqtt": "npx cucumber-js --format progress-bar --profile mqtt ./features-legacy",
+    "test:functional:legacy:cluster": "npx cucumber-js --format progress-bar --profile websocketCluster ./features-legacy",
     "test:functional:sdk": "npm run test:functional:sdk:http && npm run test:functional:sdk:websocket",
```

This should be renamed as well since it's confusing because we might think that we are testing the SDK features and not Kuzzle ones 
```suggestion
    "test:functional": "npm run test:functional:http && npm run test:functional:websocket",
```

---

Wrong change side